server/src/interfaces/notification.interface.ts

```diff
@@ -30,7 +38,35 @@ export interface WelcomeEmailProps {
   password?: string;
 }
 
-export type EmailRenderRequest = { template: EmailTemplate.WELCOME; data: WelcomeEmailProps };
+export interface AlbumInviteEmailProps {
+  baseUrl: string;
```

Comment:
looks like it might be worthwhile to have a BaseEmailProps interface that has baseUrl, since it will probably be used in each template.